core/workspace_comment_svg.js

```diff
@@ -414,26 +415,9 @@ WorkspaceCommentSvg.prototype.moveToDragSurface = function() {
   // This is in workspace coordinates.
   const xy = this.getRelativeToSurfaceXY();
   this.clearTransformAttributes_();
-  this.workspace.blockDragSurface_.translateSurface(xy.x, xy.y);
+  this.workspace.getBlockDragSurface().translateSurface(xy.x, xy.y);
   // Execute the move on the top-level SVG component
-  this.workspace.blockDragSurface_.setBlocksAndShow(this.getSvgRoot());
-};
-
-/**
- * Move this comment back to the workspace block canvas.
- * Generally should be called at the same time as setDragging(false).
- * Does nothing if useDragSurface_ is false.
- * @param {!Coordinate} newXY The position the comment should take
- *     on on the workspace canvas, in workspace coordinates.
- * @private
- */
-WorkspaceCommentSvg.prototype.moveOffDragSurface = function(newXY) {
```

Comment:
I see that this is not getting used, but I'm a little uncomfortable removing it without knowing the original reason for leaving it around. 

Also, I would rather we removed this in a PR with a description that contains more information on why we are removing it.

Reply:
This function needs to be removed in order to fix closure-compiler errors.
Leaving it in causes `ERROR - [JSC_UNUSED_PRIVATE_PROPERTY]`.

I looked into it and found that this method was introduced in https://github.com/google/blockly/pull/1817 along with `WorkspaceCommentSvg.moveToDragSurface` and browsing the files at that commit, it seems like neither of those method was ever being called. I think it introduced initially because `BlockDragger` requires `BlockSvg` has those methods, but `BubbleDragger` implements its own version of `moveToDragSurface` and `moveOffDragSurface`.

In addition to this method, `moveToDragSurface` and `clearTransformAttributes_` is dead code. I can remove those two methods as well and see that it's never used. `moveToDragSurface` doesn't cause compile errors because it is `@package` as of https://github.com/google/blockly/pull/3335 (however, I think that change only happened because it shares a name with the method on `BlockSvg`).